cli/src/main.rs

```diff
@@ -289,11 +289,25 @@ struct Test {
 
 #[derive(Args)]
 #[command(alias = "publish")]
-/// Increment the version of a grammar
+/// Print or increment the version of a grammar
 struct Version {
-    #[arg(num_args = 1)]
     /// The version to bump to
+    #[arg(
+        default_value = "0.0.0",
+        long_help = "When present, the version to bump to\n\
+                     When absent with no flags present, prints the current version\n\
+                     When absent with flags present, bumps the current version"
+    )]
     pub version: SemverVersion,
+    /// Bump the patch version
+    #[arg(long)]
+    pub bump: bool,
+    /// Bump the minor version
+    #[arg(long)]
+    pub bump_minor: bool,
+    /// Bump the major version
+    #[arg(long)]
+    pub bump_major: bool,
```

Comment:
Instead of 3 bool flags, this should be one argument, `bump`, that takes an enum, of which the enum has the major/minor/patch variants. Then, you can make patch the default variant.

Reply:
I cannot get Rust to make "patch" the default variant.  The "Optional" wrapper is not playing along.

I'm happy to leave it as explicit though.  You then say

- `tree-sitter version` to display the current version
- `tree-sitter version 1.2.3` to set it directly (current behaviour, which I do not want to change)
- `tree-sitter version --bump patch` to bump the patch.
- `tree-sitter version --bump minor` to bump the minor; etc.

Reply:
Works for me; the added default is only very minor QoL.